Simplified/AxisService/NYPLAxisService.swift

```diff
@@ -216,37 +263,35 @@ class NYPLAxisService: NSObject {
   // MARK: - Helper Methods
   private func downloadItem(from url: URL, at writeURL: URL) {
     
-    // TODO: OE-62: Axis code might need specific requirements for caching and
-    // using the shared network executor will result in whatever we choose to
-    // user there.
+    guard shouldContinue else {
+      return
+    }
     
-    NYPLNetworkExecutor.shared.GET(url) { (result) in
+    downloader?.downloadItem(from: url) { (result) in
```

Comment:
retain cycle, I think, because downloader retains the closure

Reply:
Fixed